src/FileUploader.ts

```diff
@@ -3,21 +3,17 @@ import { UploadFile } from './types/UploadFile'
 import { ErrorResponse, LoginRequiredError } from './types/errors'
 import { RequestClient } from './request/RequestClient'
 import { ServerType } from '@sasjs/utils/types'
-import SASjs from './SASjs'
-import { Server } from 'https'
 import { SASjsConfig } from './types'
-import { config } from 'process'
 
 export class FileUploader {
-  constructor(
-    private sasjsConfig: SASjsConfig,
-    private jobsPath: string,
-    private requestClient: RequestClient
-  ) {
-    if (this.sasjsConfig.serverUrl) isUrl(this.sasjsConfig.serverUrl)
-  }
+  constructor(private jobsPath: string, private requestClient: RequestClient) {}
```

Comment:
Removed `sasjsConfig`, as private member of `FileUploader`. We already have sasjsConfig in `SASjs.ts`

Looking at the usage of file uploading in fileUploader repo, calling it through `SASjs` instance
- [usage 1](https://github.com/sasjs/fileuploader/blob/f7959bef0f192fd05b6ba2074dbea8758b0f6156/src/scripts.js#L64)
- [usage 2](https://github.com/sasjs/fileuploader/blob/f7959bef0f192fd05b6ba2074dbea8758b0f6156/src/scripts.js#L91)